apps/sms/views/conversation/style/message.css

```diff
@@ -35,11 +40,15 @@
 }
 
 .message.incoming .bubble {
+  background: #39c3c2;
+}
+html[dir="ltr"] .message.incoming .bubble {
   float: left;
-
   border-radius: 0 1.5rem 1.5rem 1.5rem;
-
-  background: #39c3c2;
+}
+html[dir="rtl"] .message.incoming .bubble {
```

Comment:
I think we're at a point where we need to optimize from different directions -- and fixing some of our awful selectors is one of the directions.